plugins/chat/spec/queries/chat/messages_query_spec.rb

```diff
@@ -71,6 +71,11 @@
       end
     end
 
+    it "limits results of paginated query when page_size is not set" do
```

Comment:
This and the other fix are so strange because it's not supposed to do the paginated query if target_message_id is provided...trying it out locally

Reply:
Oh wait I see it now 🤦